### PR TITLE
[win32] support high frame rates

### PR DESCRIPTION
**Pull request by jonahwilliams**
On the win32 embedding, asking dwm for the frame rate and use this to schedule frames instead of only using 60 hz. For UWP this remains 60 hz.

Increase the default timer resolution for win 10 and greater. I think this restriction could be loosened a bit in the future, but this is a fairly conservative choice.

### CODE REVIEW

File: fml/platform/win/message_loop_win.cc [hunk 1]
```diff
@@ -4,13 +4,31 @@
 
 #include "flutter/fml/platform/win/message_loop_win.h"
 
+#include <VersionHelpers.h>
+#include <timeapi.h>
+
 #include "flutter/fml/logging.h"
 
+const uint32_t kHighResolutionTimer = 1;  // 1 ms
+const uint32_t kLowResolutionTimer = 15;  // 15 ms
```
Comment (dnfield) [inline fml/platform/win/message_loop_win.cc:13]: constexpr

**Review by dnfield — APPROVED** [review]
LGTM with nit

File: fml/platform/win/message_loop_win.cc (hunk 1 above)
Comment (jonahwilliams) [reply fml/platform/win/message_loop_win.cc]: done

**Comment by clarkezone** [comment]
AWESOME!

**Pull request merged**